### PR TITLE
Price drop trigger

### PR DESCRIPTION
**Pull request by BhargavJethwa**
Added price drop trigger feature.

### CODE REVIEW

**Comment by shahrk** [comment]
Has this been properly tested?

**Comment by shahrk** [comment]
I'd love to see some screenshots of the price trigger notification in action

File: code/BestBuyScraper.py [hunk 1]
```diff
@@ -49,7 +49,7 @@ def check_stock_price(self, url):
         try:
             page = requests.get(url, headers=headers, timeout=5)
             # parsing the content of the page
-            soup = BeautifulSoup(page.text, "lxml")
+            soup = BeautifulSoup(page.text, "html.parser")
```
Comment (shahrk) [inline code/BestBuyScraper.py:52]: What was the exact issue here?

File: code/GUI.py [hunk 2]
```diff
@@ -23,8 +23,8 @@
 import plyer
 import os
 import Tracker
-import pystray
-from pystray import MenuItem as item, Menu as menu
+import pystray._win32
```
Comment (shahrk) [inline code/GUI.py]: Are we sure this will not break the app for mac?

Comment (BhargavJethwa) [reply code/GUI.py]: Yes, it might, will fix that in the next commit.

File: code/BestBuyScraper.py (hunk 1 above)
Comment (BhargavJethwa) [reply code/BestBuyScraper.py:52]: lxml has some other dependencies which were not installed. So I changed it to another parser that is used for AmazonScrapper and it also works here. I have manually tested this.

**Comment by BhargavJethwa** [comment]
> I'd love to see some screenshots of the price trigger notification in action




Since it was not possible to capture a price drop while the app is running, so need to come up with some strategy on how it can be done(during the demo). However, I changed the trigger to notify even if the price remains the same, in order to see the trigger in action.

**Comment by BhargavJethwa** [comment]
Also, have changed the notification from


to

**Comment by shahrk** [comment]
Looks good!

**Comment by mithildave** [comment]
Looks really well, Bhargav!

**Pull request merged**